cypress/integration/Motives.spec.js

```diff
@@ -21,13 +21,13 @@ context('Motives Intro', () => {
 
   it('can navigate to motive selection', () => {
     cy.get('button').contains('Weiter zur Auswahl').click()
-    cy.url().should('contain', '/motives/selection')
+    cy.url().should('contain', '/motives/entry')
```

Comment:
You could test if the url does not only contain '/motives/entry' but is exactly '/motives/entry':

```suggestion
    cy.url().should('eq', '/motives/entry')
```

Reply:
I tried that, but it didn't work, because the full URL is something like"localhost:3000/motives/entry". I could add the baseurl to it, but it makes it more complicated. In my case it should be enough to use "contain".